src/bindings/python/flux/job.py

```diff
@@ -98,6 +98,32 @@ def wait(flux_handle, jobid=lib.FLUX_JOBID_ANY):
     return status
 
 
+# Due to subtleties in the python bindings and this call, this binding
+# is more of a reimplementation of flux_job_list().  Subtle issues
+# are:
+#
+# - Python Future class does not have get()
```

Comment:
It does; it's just not explicitly written.  Calling `.get()` on any future will call `flux_future_get` thanks to some magic that @trws wrote in the `Wrapper` class.

Reply:
ahhh, what I meant was `.get_str()` which is only in `rpc.py`